packages/p2p/src/stores/order-store.js

```diff
@@ -64,8 +71,46 @@ export default class OrderStore {
             p2p_order_confirm: 1,
             id,
         }).then(response => {
-            if (response && response.error) {
-                order_details_store.setErrorMessage(response.error.message);
+            if (response) {
+                if (response.error) {
+                    if (response.error.code === 'OrderEmailVerificationRequired') {
+                        clearTimeout(wait);
+                        const wait = setTimeout(() => this.setIsEmailVerificationModalOpen(true), 250);
+                    } else if (
+                        response?.error.code === 'InvalidVerificationToken' ||
+                        response?.error.code === 'ExcessiveVerificationRequests'
+                    ) {
```

Comment:
```suggestion
    else if (['InvalidVerificationToken', 'ExcessiveVerificationRequests'].includes(response.error.code))
```

Reply:
Or maybe:
```suggestion
if (response) {
  if (response.error) {
    switch (response.error.code) {
      case "OrderEmailVerificationRequired":
        clearTimeout(wait);
        const wait = setTimeout(
          () => this.setIsEmailVerificationModalOpen(true),
          250
        );
        break;
      case "InvalidVerificationToken":
      case "ExcessiveVerificationRequests":
        clearTimeout(wait);
        this.setVerificationLinkErrorMessage(response.error.message);
        const wait = setTimeout(
          () => this.setIsInvalidVerificationLinkModalOpen(true),
          230
        );
        break;
      case "ExcessiveVerificationFailures":
        clearTimeout(wait);
        this.setVerificationLinkErrorMessage(response.error.message);
        const wait = setTimeout(
          () => this.setIsEmailLinkBlockedModalOpen(true),
          230
        );
      default:
        order_details_store.setErrorMessage(response.error.message);
    }
  }
}
```

Reply:
As for the Array.includes, it's a smaller statement but my (future) plan is to create a const file with all error codes, that would be cleaner. 

For the switch case, if you have any compelling reasons for me to change this, I'm all ears but I think it would work the same so I don't see the point in changing it.